pkg/resources/config.go

```diff
@@ -0,0 +1,44 @@
+package resources
+
+import (
+	"time"
+
+	arkv1 "github.com/heptio/ark/pkg/apis/ark/v1"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func Config(
+	namespace string,
+	cloudProviderName string,
```

Comment:
I would have 1 for backup storage & 1 for pv storage - the configs are different